scripts/common/host-packages.sh

```diff
@@ -164,6 +164,13 @@ EOF
     yum makecache --disablerepo=* --enablerepo=kurl.local
 
     # shellcheck disable=SC2086
```

Comment:
could you copy this comment so that it's present for both the openssl and containerd.io `if` lines?
```
# shellcheck disable
```
only affects the line immediately after it

Reply:
(also, we should really look at improving our shellcheck checking in general and maybe add it to CI)

Reply:
done